processing/api.py

```diff
@@ -128,6 +127,13 @@ def add_track_tag(self, recording, track_id, prediction, data=""):
             return r.json()["trackTagId"]
         raise IOError(r.text)
 
+    def get_track_info(self, recording_id):
+        r = requests.get(
+            self.file_url + "/{}/tracks".format(recording_id),
+        )
```

Comment:
I think you can use this now. I nicer string formatting I think. https://realpython.com/python-f-strings/#simple-syntax